Summers_James/app_base.htm

```diff
@@ -0,0 +1,150 @@
+                        </div>
+                        <!--/.Second column-->
+            
+                        <!--Third column-->
+                        <div class="col-md-4 col-lg-3">
+                            <h5 class="title mb-4 font-bold footer-title">Address</h5>
+                            <!--Info-->
+                            <p><i class="fa fa-home mr-3"></i> New York, NY 10012, US</p>
+                            <p><i class="fa fa-envelope mr-3"></i> info@example.com</p>
+                            <p><i class="fa fa-phone mr-3"></i> + 01 234 567 88</p>
+                            <p><i class="fa fa-print mr-3"></i> + 01 234 567 89</p>
+                        </div>
+                        <!--/.Third column-->
+            
+                        <!--Fourth column-->
+                        <div class="col-md-2 col-lg-2 text-center">
+                            <h5 class="title mb-4 font-bold footer-title">Follow Us</h5>
+                            <!--Social buttons-->
+                            <div class="social-section mt-2 ">
+                                <!--Facebook-->
+                                <a type="button" class="btn-floating btn-small btn-fb"><i class="fa fa-facebook" style="padding-right: 20px;"><img src="${ STATIC_URL }homepage/media/images/facebook.png" style="width:30px; height: 30px;"/></i></a>
+                                <!--Twitter-->
+                                <a type="button" class="btn-floating btn-small btn-tw"><i class="fa fa-twitter"><img src="${ STATIC_URL }homepage/media/images/twitter.png" style="width:30px; height: 30px;"/></i></a>
+                                <!--Instagram-->
+                                <br><br>
+                                <a type="button" class="btn-floating btn-small btn-gplus"><i class="fa " style=" padding-right: 20px;"><img src="${ STATIC_URL }homepage/media/images/instagram.png" style="width:30px; height: 30px;"/></i></a>
+                                <!--Linked-In-->
+                                <a type="button" class="btn-floating btn-small btn-dribbble"><i class="fa fa-dribbble"><img src="${ STATIC_URL }homepage/media/images/linkedin.png" style="width:30px; height: 30px;"/></i></a>
+                            </div>
+                        </div>
+                        <!--/.Fourth column-->
+                      </div>
+                    </div>
+                    <!--/.Footer Links-->
+                
+                    <!--Copyright-->
+                    <div class="footer-copyright myfooter-copyright">
+                        <div class="container-fluid myfooter_container-fluid">
+                            <%! from datetime import datetime %>
+                             © ${ datetime.now().year } Copyright
+                        </div>
+                    </div>
+                    <!--/.Copyright-->
+                
+                </footer>
+                <!--/.Footer-->
```

Comment:
I like these commented html blocks to show where sections of the layout are, makes it easy to tell what tags are a part of what.

Reply:
👍